src/nspawn/nspawn.c

```diff
@@ -3286,6 +3463,15 @@ static int load_settings(void) {
                 }
         }
 
+        if ((arg_settings_mask & SETTING_NOTIFY_MODE) == 0) {
+
+                if (!arg_settings_trusted)
+                        log_warning("Ignoring NotifyMode= settings, file %s is not trusted.", p);
+                else {
```

Comment:
I don't think this should be a privileged setting. After all, if it's set incorrectly it will only affect the container itself, not really the host. the "trusted" logic here is really just there to protect the host from containers that want to do random shit with the host's network.